src/services/organization.js

```diff
@@ -52,6 +53,45 @@ module.exports = class OrganizationService {
 			})
 		} catch (error) {
 			console.log(error)
+			throw error
+		}
+	}
+
+	static async createOrgExtension(eventBody) {
+		try {
+			const extensionData = {
```

Comment:
@joffinjoy can we use the eventBody Object to upsert, instead of accessing and storing the fields